pkg/storaged/fsys-tab.jsx

```diff
@@ -159,7 +159,10 @@ export function check_mismounted_fsys(client, path, enter_warning) {
         else if (is_mounted && opt_noauto && !opt_noauto_intent && !opt_systemd_automount)
             type = "no-mount-on-boot";
     } else if (other_mounts.length > 0) {
-        type = "mounted-no-config";
+        // We don't complain about the rootfs, it's probably
+        // configured somewhere else, like in the bootloader.
+        if (other_mounts[0] != "/")
```

Comment:
Is it not possible that there are multiple entries, and / is only the second or later entry?

Reply:
Yes maybe, but almost all of the current Cockpit storage page only looks at the first entry, since the UDisks2 API can only unmount that one anyway, and the world is just a simpler place if we close our eyes to multiple mount points... :-)

Reply:
Ack, thanks! Can you please rebase this PR, so that we get over the "protection checks" failure? I'll approve then.

Reply:
Done!